server/src/utils/request.ts

```diff
@@ -0,0 +1,101 @@
+import axios from 'axios';
+import FormData from 'form-data';
+import logger from '@shared/Logger';
+
+export async function post(endpoint_url: string, data: FormData, extra: any) {

```

Comment:
```suggestion
export async function post(endpoint_url: string, data: FormData, headers: any) {

```
We should avoid using the `any` typing whenever possible.. not sure if the headers have a generic typing or not. If so you should be able to get it from the response type of this function: `form.getHeaders()`

Reply:
Can you switch from snake -> camel?